rsts/howto/managing_customizable_resources.rst

```diff
@@ -47,6 +49,15 @@ in the flyteadmin config are used.
 
 The override values from the database are assigned at execution time.
 
+To update individual project-domain attributes, use the following as an example:
+
+.. code-block:: bash
+
+    curl --request PUT 'https://flyte.company.net/api/v1/project_domain_attributes/projectname/staging' \
```

Comment:
can this be a flyte-cli command?

Reply:
it can but it's not implemented yet and I don't know if we should if it'll be superseded by flyte-ctl anyways

Reply:
aah got it, just write that as a note then